target/linux/ramips/dts/WIZFI630S.dts

```diff
@@ -0,0 +1,196 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "wiznet,wizfi630s", "mediatek,mt7628an-soc";
+	model = "WIZnet WizFi630S";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+        aliases {
+                led-boot = &led_run;
+                led-failsafe = &led_run;
+                led-running = &led_run;
+                led-upgrade = &led_run;
+        };
+
+	gpio-leds {
```

Comment:
@7tobias You've some white space issue here. While at it could you please change this to generic name `leds` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation